src/frontend/Player/createVideojsPlayer.spec.tsx

```diff
@@ -376,4 +398,132 @@ describe('createVideoJsPlayer', () => {
 
     expect(player.cache_.initTime).toEqual(10);
   });
+
+  it('sends attendance for a student watching a live', async () => {
+    mockGetDecodedJwt.mockReturnValue(ltiStudentTokenMockFactory());
+    mockIsMSESupported.mockReturnValue(true);
+    const video = videoMockFactory({
+      urls: {
+        manifests: {
+          hls: 'https://example.com/hls',
+        },
+        mp4: {},
+        thumbnails: {},
+      },
+      live_state: liveState.RUNNING,
+    });
+    const { container } = render(
+      wrapInIntlProvider(
+        <VideoPlayer
+          video={video}
+          playerType={'videojs'}
+          timedTextTracks={[]}
+        />,
+      ),
+    );
+
+    await waitFor(() =>
+      // The player is created
+      expect(mockCreatePlayer).toHaveBeenCalled(),
+    );
+
+    const videoElement = container.querySelector('video');
+
+    const player = createVideojsPlayer(videoElement!, jest.fn(), video, 'en');
+
+    expect(player.options_.liveui).toBe(true);
+
+    expect(mockPushAttendance).not.toHaveBeenCalled();
+    // to initialize the video for a live
+    player.qualityLevels().trigger('change');
+    Date.now = jest.fn(() => 1651732370000);
+    jest.runOnlyPendingTimers();
+    expect(mockPushAttendance).toHaveBeenCalledWith(
+      {
+        1651732370000: {
+          fullScreen: player.isFullscreen(),
+          muted: player.muted(),
+          player_timer: player.currentTime(),
+          playing: !player.paused(),
+          timestamp: 1651732370000,
+          volume: player.volume(),
```

Comment:
you should not compute the expected values but hardcode them instead to be sure this is really what you want.